setup.py

```diff
@@ -1,16 +1,25 @@
 from setuptools import setup, find_packages
 
+requirements = [
```

Comment:
Either requirements.txt should be removed, or the versions mentioned in requirements.txt should be implemented in this list.

Reply:
Both lists should exists.
requirements.txt describes the required packages if you want to setup a environment specifically for openspoor. These packages are known to work. You will use this if you for example want to setup a Docker container specific for openspoor or want to develop openspoor.
In the setup.py, you list the required packages with little constraints. Only known versions of packages that don't work are excluded (e.g. pyproj>3). The setup.py is used when you want to install openspoor in an existing environment (hopefully soon through "pip install openspoor"). When installing in an existing environment, you don't want to have strict version numbers from requirements.txt, because than the entire environment needs to be changed for openspoor, leading to incompatibilities with other packages in that environment that maybe don't work with that specific version of openspoors requirements.
So you use strict version numbers in requirements.txt, resulting in a known working environment, and you use loose versions in setup.py, to achieve compatibility with other packages in an (existing) environment